ee/WFLY-14061_EAP7-1366_JavaEE-to-JakartaEE-refs.adoc

```diff
@@ -0,0 +1,58 @@
+= Replace JavaEE references with corresponding JakartaEE ones
+:author:            Alessio Soldano
+:email:             asoldano@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+As WildFly is JakartaEE 8 certified since version 18 (https://github.com/wildfly/certifications/blob/EE8/WildFly_18.0.0.Final/jakarta-full-platform.adoc) and JakartaEE 9 implementation is in progress, we want to properly position the project as a JakartaEE implementation more than a JavaEE one.
+
+== Issue Metadata
+
+=== Issue
+
+* {issue-base-url}/EAP7-1366 [EAP7-1366]
+* {issue-base-url}/WFLY-14061 [WFLY-14061]
+
+=== Related Issues
+
+* {issue-base-url}/EAP7-1193 [EAP7-1193]
+
+=== Dev Contacts
+
+* mailto:asoldano@redhat.com[Alessio Soldano]
+
+=== QE Contacts
+
+* Jan Blizňák
+
+=== Testing By
+
+[ ] Engineering
+
+[x] QE
+
+=== Affected Projects or Components
+
+* WildFly
+
+=== Other Interested Projects
+
+* None
+
+== Requirements
+
+=== Hard Requirements
+
+* We need to replace any occurrence of JavaEE terms with the corresponding JakartaEE one. This applies to javadoc, code comments (possibly), community documentation, quickstarts, examples, etc.
+
+== Test Plan
+
+* This feature request shouldn't imply any functional change to the code. Hence we should just ensure no regression is caused in the existing testsuites.
```

Comment:
Just to be sure, we already moved away from JavaEE APIs everywhere to be save to state that?

Reply:
Yes, we did; in WildFly 18 we ensured that all the jboss spec api forks that we consume are derived from JakartaEE api artifacts (https://issues.redhat.com/browse/WFLY-12300)